borrowing/urls.py

```diff
@@ -1,9 +1,14 @@
 from django.urls import path
 
-from borrowing.views import BorrowingCreateView, BorrowingListView
+from borrowing.views import (
+    BorrowingListView,
+    BorrowingDetailView,
+    BorrowingCreateView
+)
 
 urlpatterns = [
     path("", BorrowingListView.as_view(), name="borrowing-list"),
```

Comment:
Can we use one route, but chose different serializers based on self.action and get_serializer_class?